src/pushprocessor.ts

```diff
@@ -162,7 +168,7 @@ export class PushProcessor {
         if (!newRules) newRules = {} as IPushRules;
         if (!newRules.global) newRules.global = {} as PushRuleSet;
         if (!newRules.global.override) newRules.global.override = [];
-        if (!newRules.global.override) newRules.global.underride = [];
+        if (!newRules.global.underride) newRules.global.underride = [];
```

Comment:
This seems to have been a separate, long-standing bug?

Reply:
Yeah, how exciting. What would the symptoms have been? Maybe we can find an issue to close....

Reply:
It was added in #2873, but that also added (on line 191) a `newRules.global.underride ?? []`, likely to solve the same problem? 🤷 (I don't think this would have a user-visible bug.)